app/assets/javascripts/tophat_space.coffee

```diff
@@ -0,0 +1,5 @@
+$('document').ready ->
+  #checks if tophat exists 
+  tophatExist = document.getElementsByClassName("th-wrapper")
```

Comment:
Sorry I didn't notice this before, but you should make this call with JQuery, which you are using above and below

Reply:
yea, that is a good point. https://stackoverflow.com/questions/23648196/converting-document-getelementsbyclassname-to-jquery has an example of this.